frontend/src/hooks/useExceptionHandler.jsx

```diff
@@ -18,6 +18,21 @@ const useExceptionHandler = () => {
         duration: duration,
       };
     }
+    if (err.code === "ERR_NETWORK" && !navigator.onLine) {
+      return {
+        type: "error",
+        content: "Please check your internet connection.",
+        title: title,
+        duration: duration,
+      };
+    } else if (err.code === "ERR_CANCELED") {
+      return {
+        type: "error",
+        content: "Request has been canceled.",
+        title: title,
+        duration: duration,
+      };
+    }
```

Comment:
```suggestion  
    const getErrorMessage = (err, title, duration) => {
  const errorMessages = {
    ERR_NETWORK: "Please check your internet connection.",
    ERR_CANCELED: "Request has been canceled.",
  };

  if (errorMessages[err.code]) {
    return {
      type: "error",
      content: err.code === "ERR_NETWORK" && !navigator.onLine ? errorMessages.ERR_NETWORK : errorMessages[err.code],
      title,
      duration,
    };
  }

  return null; // or handle other error cases
};

```

Reply:
Check and apply this change if seems okay. @jagadeeswaran-zipstack

Reply:
I don't think this can be used because adding more error codes will make the content condition more complicated. If you think otherwise please let me know.